packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverCarouselNavIcon/TeachingPopoverCarouselNavIcon.types.ts

```diff
@@ -0,0 +1,34 @@
+import { PopoverContextValue } from '@fluentui/react-popover';
+import type { ComponentState, ComponentProps, Slot } from '@fluentui/react-utilities';
+
+export type TeachingPopoverCarouselNavIconSlots = {
+  /**
+   * The root slot of the TeachingPopoverCarouselNavIcon.
+   * Default html element is div
+   */
+  root: NonNullable<Slot<'div'>>;
+  /**
+   * Nav icon set as an ARIA compliant button
+   */
+  navButton: NonNullable<Slot<'button'>>;
```

Comment:
I have to say that naming is weird there. IMO it should be `TeachingPopoverCarouselNavButton` as icons are non-interactive items